templates/universal-standard/script/universalresults.hbs

```diff
@@ -82,8 +82,9 @@ ANSWERS.addComponent("UniversalResults", Object.assign({}, {
     mapConfig: Object.assign({
       apiKey: HitchhikerJS.getDefaultMapApiKey("{{ mapConfig.mapProvider }}")
     },
-    {"providerOptions": {
-      "scrollZoom": false
+    {
+      "providerOptions": {
+        "scrollZoom": false
     }},
```

Comment:
new line between the curly braces here too

Reply:
like this?
```
{
   "providerOptions": {
      "scrollZoom": false
   }
},
```

Reply:
yes